tutorials/_index.md

```diff
@@ -4,6 +4,7 @@ weight = 2
 sort_by = "weight"
 template = "sections/docs/chapters.html"
 aliases = ["docs/learn/"]
+insert_anchor_links = "right"
```

Comment:
These _index pages have been moved - not sure if git will know where to put them since they all have the same name. There are also several new _index pages that need to have this line added.

Reply:
My git-fu is not so good, so I don't know how exactly this should be done. Would I need to merge urbit/docs into this branch and then set the front matter properties again?

Reply:
Okay so I tried that out and I think it worked. Which `_index.md` files do you want it enabled on? With the old docs scheme, I had it enabled on the tutorial sections because they were long documents with headers that would most likely be linked. I'm not as familiar with the new organization

Reply:
Just the tutorial ones you did for now will be fine. We've been splitting up longer pages elsewhere into separate pages.

Reply:
Ok so I added the anchors to any section where I saw long form content that would benefit from it.